packages/node_modules/@node-red/editor-client/locales/de/editor.json

```diff
@@ -82,8 +94,9 @@
             "palette": {
                 "show": "Palette anzeigen"
             },
+            "edit": "Bearbeiten",
             "settings": "Einstellungen",
-            "userSettings": "Einstellungen",
+            "userSettings": "Benutzereinstellungen",
```

Comment:
settings == Einstellungen
user == Benutzer

userSettings == Benutzereinstellungen

This makes sense if (yet only if) operators should be able to distinguish `settings` from `userSettings`. Otherwise I'd vote to keep `Einstellungen` - saving screen space without loosing context.